### PR TITLE
Make footer on product page spread itself when the menu is hidden

### LINKED ISSUE #11130: Problems in the admin side on the bottom toolbar of product pages under 1039px
Hi
if you go in a product page in the back office
try to resize the borwser's width under 1039px you will see the the bottom toolbar will not be full size
it should have ad width of 100% because the lateral menu is not visible

**Screenshots**

**Additionnal information**
PrestaShop version: 1.7.4.3
PHP version: 7

bye

### PR DESCRIPTION
**Pull request by dineshbadrukhiya**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Problems in the Back-Office on the bottom toolbar of product page.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? |  Fixes #11130
| How to test?  | See fixed ticket #11130

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12931)

### CODE REVIEW

**Comment by dineshbadrukhiya** [comment]
@matks I have updated develop branch on my computer as you have guided in #12904 . So can you please compile the assets for me?

**Comment by matks** [comment]
Here you go 😄

**Comment by matks** [comment]
@dineshbadrukhiya I tested your PR but I dont see the expected result. When I load Product page, I dont see `page-sidebar-closed` CSS class. Maybe it got removed but you did not see it as your `develop` branch was outdated. Can you check on your computer ?

**Comment by dineshbadrukhiya** [comment]
> @dineshbadrukhiya I tested your PR but I dont see the expected result. When I load Product page, I dont see `page-sidebar-closed` CSS class. Maybe it got removed but you did not see it as your `develop` branch was outdated. Can you check on your computer ?

As I have checked with the 1.7.5.1, `page-sidebar-closed` CSS class is applied when menu collapse in the navbar. 

One more thing as I have checked with your compiled css assests, there is no css applied which I have added in the `admin-dev/themes/new-theme/scss/pages/_product_page.scss` file.

**Comment by matks** [comment]
> As I have checked with the 1.7.5.1, `page-sidebar-closed` CSS class is applied when menu collapse in the navbar.

That was the information I was missing 😉 thanks

> One more thing as I have checked with your compiled css assests, there is no css applied which I have added in the `admin-dev/themes/new-theme/scss/pages/_product_page.scss` file.

I promise it's there, but it's rewritten by webpack.

So I checked again with your info, I confirm that the following scenario is fixed:

### Before the PR

If I go on Edit a Product page and I collapse the left menu bar, the bottom toolbar size does not change.

### With the PR

If I go on Edit a Product page and I collapse the left menu bar, the bottom toolbar size adapts ✅ 

So you have a fixed a bug (👏) ... but it seems to me that it's not the same bug as the one described in https://github.com/PrestaShop/PrestaShop/issues/11130 . To my understanding, in https://github.com/PrestaShop/PrestaShop/issues/11130 the bug report is rather "If I go on Edit a Product page and I reduce the size of the browser window, at some point (1039px it seems) the left menu bar collapses but the bottom toolbar size does not change."

**Comment by matks** [comment]
@dineshbadrukhiya Can you confirm ? I ask so we can tell to QA whether the bug fixed is https://github.com/PrestaShop/PrestaShop/issues/11130 or another one.

**Comment by dineshbadrukhiya** [comment]
> To my understanding, in #11130 the bug report is rather "If I go on Edit a Product page and I reduce the size of the browser window, at some point (1039px it seems) the left menu bar collapses but the bottom toolbar size does not change."

I think commit `2ea12a6` will also solve #11138 

Do you have an idea about commit `2a37463` as I have no idea about it why it's showing here because I haven't changed anything in that file.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Okay, your initial commit @dineshbadrukhiya was fixing an issue, but another one was remaining when the menu disappeared completely. I added changes to this PR and commited the assets in order to make the review going on.

**Review by Quetzacoalt91 — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/_product_page.scss [hunk 1]
```diff
@@ -759,6 +762,14 @@
   }
 }
 
+.page-sidebar-closed:not(.mobile) {
+  .product-page {
+      .product-footer {
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/pages/_product_page.scss]: indentation problem here :)

Comment (Quetzacoalt91) [reply admin-dev/themes/new-theme/scss/pages/_product_page.scss]: Fixed

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/scss/pages/_product_page.scss]: Danke

**Comment by Quetzacoalt91** [comment]
Is there something wrong @dineshbadrukhiya? Now that we took the time to make your PR valid and approved, I don't understand why it's now closed.

**Comment by dineshbadrukhiya** [comment]
@Quetzacoalt91  Even I don't know how it was closed. For now I have re-opened it.

**Comment by Quetzacoalt91** [comment]
Thank you @dineshbadrukhiya

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @dineshbadrukhiya

**Pull request merged**